executor/seqtest/prepared_test.go

```diff
@@ -21,33 +21,31 @@ import (
 	"github.com/pingcap/tidb/executor"
 	"github.com/pingcap/tidb/metrics"
 	plannercore "github.com/pingcap/tidb/planner/core"
+	"github.com/pingcap/tidb/session"
 	"github.com/pingcap/tidb/types"
+	"github.com/pingcap/tidb/util/kvcache"
 	"github.com/pingcap/tidb/util/testkit"
 	dto "github.com/prometheus/client_model/go"
 )
 
 func (s *seqTestSuite) TestPrepared(c *C) {
 	orgEnable := plannercore.PreparedPlanCacheEnabled()
-	orgCapacity := plannercore.PreparedPlanCacheCapacity
```

Comment:
after this change, it seems many of the tests under `seqTestSuite` can be run in parallel now. Shall we reorganize `seqTestSuite` and `seqTestSuite1` to caange one of them to `Suite` so we can gain better performance

Reply:
but `orgEnable := plannercore.PreparedPlanCacheEnabled()` is still a global variable it changed, I don't know if it will affect other test cases.

Reply:
ok, they should stay serial then